### DISCUSSION TITLE
Recent update broke LSP rename

### DISCUSSION THREAD

**Discussion by clinta** [LazyVim/LazyVim#5082] [Q&A]
My config has been static for quite a while, but after some recent update that I installed today, my LSP rename is no longer working and opens an unnamed buffer instead of renaming anything. I've tried with multiple LSPs and languages and it happens in every one. Any ideas?

https://github.com/user-attachments/assets/2b21b2f8-786d-47f0-be70-2b6bdf7c64ef

**Comment by folke** [comment]
Can't reproduce this, but make sure to update snacks and try again.
If the problem persists, please open an issue in the snacks repo with a repro.

**Comment by clinta** [comment]
You did something that fixed it. I was working on repro steps in a fresh docker container to demonstrate it, I had them documented, and just a few minutes ago I ran through the repro steps, snacks had updates and the issue no longer happened. Updated my own machine and it's all now working.

**Reply by folke** [reply]
haha, great :)
